products/msexchange.md

```diff
@@ -11,22 +11,22 @@ releaseDateColumn: true
 
 releases:
 -   releaseCycle: "2019"
-    releaseLabel: "2019 CU13 SU2V2"
+    releaseLabel: "2019 CU13 SU3"
     releaseDate: 2018-10-22
     support: 2024-01-09
     eol: 2025-10-14
-    latest: "15.2.1258.25"
-    latestReleaseDate: 2023-08-15
-    link: https://support.microsoft.com/help/5030524
+    latest: "15.02.1258.027"
```

Comment:
```suggestion
    latest: "15.2.1258.27"
```

Minor change. Version from https://support.microsoft.com/en-us/topic/description-of-the-security-update-for-microsoft-exchange-server-2019-and-2016-october-10-2023-kb5030877-ec769ff1-f60f-411e-a7ed-b63b42a686eb, more similar to the previous versions.

Reply:
Thx for quick fix, I copied the long instead of the short format version...